src/uct/api/uct.h

```diff
@@ -693,11 +693,17 @@ ucs_status_t uct_iface_get_address(uct_iface_h iface, uct_iface_addr_t *addr);
  * it does not detect issues such as network mis-configuration or lack of connectivity.
  *
  * @param [in]  iface      Interface to check reachability from.
- * @param [in]  addr       Address to check reachability to.
+ * @param [in]  dev_addr   Device address to check reachability to.
+ * @param [in]  ep_addr    Remote endpoint address to check reachability to.
+ * @param [in]  iface_addr Interface address to check reachability to.
+ *
+ * @note One or more addresses can be NULL depends on interface specific.
+ *       If all addresses are NULL the return value will be 0.
```

Comment:
yep, lets remove the `ep_addr` and keep only `dev_addr` and `iface_addr`:
* `dev_addr` may be NULL if `iface_attr.dev_addr_len == 0`, and has to be non-NULL otherwise.
* `iface_addr` may be NULL if `iface_attr.iface_addr_len == 0`, and has to be non-NULL otherwise.